pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -612,52 +612,17 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 
```

Comment:
I wonder if sharing these objects between test cases is intended? Adding t.Parallel() and testing with -race would probably yield an error.